package.json

```diff
@@ -4,54 +4,35 @@
   "private": true,
   "type": "module",
   "scripts": {
-    "build:clean": "turbo run build:clean",
-    "build": "turbo run build:clean; turbo run build --concurrency=30",
-    "watch": "turbo run watch --concurrency=50 --continue",
-    "gallery": "turbo run dev:gallery",
-    "laboratory": "turbo run dev:laboratory",
-    "demo": "turbo run dev:demo",
-    "examples": "turbo run dev:example",
-    "build:gallery": "turbo run build:clean; turbo run build:gallery",
-    "build:laboratory": "npm run build; turbo run build:laboratory",
-    "build:demo": "turbo run build:clean; turbo run build:demo",
-    "build:examples": "turbo run build:clean; turbo run build:examples",
-    "test": "turbo run test",
+    "build:clean": "pnpm --if-present -r clean",
+    "build": "pnpm --if-present --workspace-concurrency=30 -r build",
+    "watch": "pnpm --if-present --workspace-concurrency=50 -r watch",
+    "gallery": "pnpm --filter=./apps/gallery dev",
+    "laboratory": "pnpm --filter=./apps/laboratory dev",
+    "demo": "pnpm --filter=./apps/gallery dev",
+    "examples": "pnpm --filter=./examples -r dev",
+    "build:packages": "pnpm --filter=./packages/* build",
+    "build:gallery": "pnpm build; pnpm --filter=./apps/gallery build",
+    "build:laboratory": "pnpm build; pnpm --filter=./apps/laboratory build",
+    "build:demo": "pnpm build; pnpm --filter=./apps/demo build",
+    "build:examples": "pnpm build; pnpm --filter=./examples -r build",
+    "test": "pnpm -r test --if-present",
     "coverage:merge": "node ./scripts/coverage-merge.js",
     "coverage:summary": "node ./scripts/coverage-summary.js",
-    "typecheck": "turbo run typecheck",
-    "lint": "turbo run lint",
+    "typecheck": "pnpm --if-present -r typecheck",
```

Comment:
why `--if-present` ?

Reply:
because not every package in the workspace has that command

Reply:
Shouldn't the turborepo already do this conditional run? If there is no script in a package, it's just not running as I know, prob something specific to pnpm?

Reply:
This is specific to `pnpm`